python_bindings/tests/end-to-python-end-test/other_two_player_game_test.py

```diff
@@ -0,0 +1,636 @@
+# flake8: noqa ascci drawing makes a lot errors
+# mypy: ignore-errors
+""""""
```

Comment:
what is this `""""""`?

Reply:
Empty docstring - kinda looks as if the author's IDE (or some plugin they use) is enforcing modules to have docstring. If that's the case then this is pylint-level of opinionated and we definitely wouldn't want to follow its suggestions in that regard (we explicitly chose not to use pylint earlier on in the project).

Reply:
Because it was counted as module docstring, and the drawing is more confusing, because everything is in italycs :/